### ISSUE TITLE
Cannot load any swf (unsupported feature VERTEX_WRITABLE_STORAGE)

### ISSUE THREAD

**Issue by tim78245** [ruffle-rs/ruffle#8981]
### Describe the bug

When attempting to use nightly release [2022/11/27](https://github.com/ruffle-rs/ruffle/releases/tag/nightly-2022-11-27) or later on any file I get the following:

```
$ ./ruffle file.swf 
MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete
[2023-01-01T00:00:00Z ERROR wgpu::backend::direct] Error in Adapter::request_device: unsupported features were requested: VERTEX_WRITABLE_STORAGE
Error: Couldn't create wgpu rendering backend

Caused by:
    Requesting a device failed
```

When using nightly release [11/26/2022](https://github.com/ruffle-rs/ruffle/releases/tag/nightly-2022-11-26) or earlier the problem does not occur and the files play fine.

### Expected behavior

Expected behavior is for ruffle to play as it did in nightly 2022/11/26 and earlier

### Affected platform

Desktop app

### Operating system

Fedora 37

**Comment by n0samu** [comment]
This is caused by #8063, which unnecessarily added the VERTEX_WRITABLE_STORAGE feature requirement. The requirement is removed again in #8262 (3f1d36cc7ed8d138c1e4ec2454aad8def340e1c2) but we should probably pull that into a separate PR so the change lands sooner.

**Comment by Lord-McSweeney** [comment]
Fixed as #8262 is merged.
